Classes/ImageService/ImgProxyImageService.php

```diff
@@ -124,6 +126,14 @@ public function processTask(TaskInterface $task): ImageServiceResult
 			);
 		}
 
+		$focusArea = $configuration['focusArea'] ?? false;
+		if ($focusArea instanceof Area && !$focusArea->isEmpty()) {
+			$focusPointX = FocusAreaUtility::calculateCenter($focusArea->getOffsetLeft(), $focusArea->getWidth());
+			$focusPointY = FocusAreaUtility::calculateCenter($focusArea->getOffsetTop(), $focusArea->getHeight());
+
+			$uri->setGravity('fp', (float)$focusPointX, (float)$focusPointY);
+		}
+
```

Comment:
Just to be in line with the other conditions, I'd prefer to have this not defining a separate variable. Also, I'm not sure that we need the type casting.

```suggestion
		if (isset($configuration['focusArea']) && !$configuration['focusArea']->isEmpty()) {
			$horizontalOffset = FocusAreaUtility::calculateCenter(
				$configuration['focusArea']->getOffsetLeft(),
				$configuration['focusArea']->getWidth(),
			);

			$verticalOffset = FocusAreaUtility::calculateCenter(
				$configuration['focusArea']->getOffsetTop(),
				$configuration['focusArea']->getHeight(),
			);

			$uri->setGravity('fp', $horizontalOffset, $verticalOffset);
		}

```